nativeio_async_wrapper.js

```diff
@@ -76,7 +88,7 @@ mergeInto(LibraryManager.library, {
       NativeIOWrapper.open(name).then(
         (fd) => {wakeUp(fd)},
         (error) => {
-          console.log('NativeIO error while opening \"', name, '\": ', error);
+          console.log('NativeIO error while opening \"'+ name + '\": ', error);
```

Comment:
I think a more readable variant is
```suggestion
          console.log(`NativeIO error while opening "${name}": ${error}`);
```

Reply:
Done.